x-pack/test/reporting_api_integration/reporting_and_security/spaces.ts

```diff
@@ -155,7 +155,7 @@ export default function ({ getService }: FtrProviderContext) {
 
     it('should complete a job of PNG export of a dashboard in non-default space', async () => {
       const downloadPath = await reportingAPI.postJobJSON(
-        `/s/non_default_space/api/reporting/generate/png`,
+        `/s/non_default_space/api/reporting/generate/pngV2`,
         {
           jobParams: `(browserTimezone:UTC,layout:(dimensions:(height:512,width:2402),id:png),objectType:dashboard,relativeUrl:'/s/non_default_space/app/dashboards#/view/3c9ee360-e7ee-11ea-a730-d58e9ea7581b?_g=(filters:!!(),refreshInterval:(pause:!!t,value:0),time:(from:!'2019-06-10T03:17:28.800Z!',to:!'2019-07-14T19:25:06.385Z!'))&_a=(description:!'!',filters:!!(),fullScreenMode:!!f,options:(hidePanelTitles:!!f,useMargins:!!t),query:(language:kuery,query:!'!'),timeRestore:!!t,title:!'Ecom%20Dashboard%20Non%20Default%20Space!',viewMode:view)',title:'Ecom Dashboard Non Default Space')`,
```

Comment:
This test should be failing, since it provides a V1 jobParams object and uses the V2 API. I guess the test is not checking for the right thing.

Reply:
jobParams should be `(browserTimezone:America/Phoenix,layout:(dimensions:(height:512,width:1724),id:preserve_layout),locatorParams:(id:DASHBOARD_APP_LOCATOR,params:(dashboardId:e35742df-db88-5bd6-9d35-732b4b04a079,preserveSavedFilters:!t,timeRange:(from:'2019-06-10T03:17:28.800Z',to:'2019-07-14T19:25:06.385Z'),useHash:!f,viewMode:view)),objectType:dashboard,title:'Ecom Dashboard Non Default Space',version:'8.10.0')`